### PR TITLE
Fix explainability dto

### PR DESCRIPTION
**Pull request by r00ta**
Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [ ] You have read the [contributors' guide](https://github.com/kiegroup/kogito-operator/blob/master/README.md#contributing-to-the-kogito-operator)
- [ ] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [ ] Pull Request contains a link to the JIRA issue
- [ ] Pull Request contains a description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](RELEASE_NOTES.md) entry regarding this change

### CODE REVIEW

File: test/features/keycloak.feature [hunk 1]
```diff
@@ -111,7 +111,10 @@ Feature: Kogito integration with Keycloak
         "modelIdentifier": {
           "resourceType": "dmn",
           "resourceId": "namespace:name"
-        }
+	},
+	"type":"LIME",
+	"inputs": {},
+	"outputs": {}
```
Comment (radtriste) [inline test/features/keycloak.feature:117]: can you please be careful with the indentation ?

**Review by radtriste — CHANGES_REQUESTED** [review]

File: test/features/keycloak.feature (hunk 1 above)
Comment (radtriste) [reply test/features/keycloak.feature:117]: woups this is GH display which is buggy ... sorry

Comment (r00ta) [reply test/features/keycloak.feature]: Actually I missed a space. Added

**Review by radtriste — APPROVED** [review]

**Comment by radtriste** [comment]
tested ok

**Review by sutaakar — APPROVED** [review]

**Pull request merged**